app/js/components/Draft.jsx

```diff
@@ -7,10 +7,21 @@ import IconButton from './button/IconButton';
 
 export class Draft extends PureComponent {
   renderDraftList = () => {
+    let draftType;
     const { draftOrders, handleDraftDiscard } = this.props;
     return draftOrders.map((order) => {
       const isPanel = !!order.set;
-      const draftType = !isPanel ? 'single' : 'panel';
+      const isOtherOrderType = !!order.type;
+
+      if (isPanel) {
+        draftType = 'panel';
+      } else {
+        draftType = 'single';
+      }
```

Comment:
instead of the `else` part, just set `draftType` to `single`

Reply:
It can't work that way. Because, if I set `draftType` to single it's going to defeat the purpose of having the conditional statement.